### ISSUE TITLE
Add language: dyff

### ISSUE THREAD

**Issue by BogdanSorlea** [github/linguist#6348]
## Language name
dyff

## URL of example repository
https://github.com/homeport/dyff

## URL of syntax highlighting grammar
TBD

## Most popular extensions
unaware of extensions, if any

## Detected language
N/A(?)

I would love to have dyff highlighting in code blocks on PRs. [Initially I went down the path](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting) of having a [highlight.js addition](https://github.com/homeport/dyff/issues/289) for this language, but then [the github docs seem to suggest liguist as the way to go](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) - to which I am opening this issue here.

**Comment by tisoi22** [comment]
Vietnam

**Comment by lildude** [comment]
> Do you know what would be the best way to achieve this?

Yup. Hit the "Contact GitHub" link on any page, or reach out in the discussions at https://github.com/community

**Comment by BogdanSorlea** [comment]
Thanks for your input.

For completion, I wrote [this community post as product feedback](https://github.com/orgs/community/discussions/52701).
